doc/input-output-reference/src/overview/group-location-climate-weather-file-access.tex

```diff
@@ -1771,7 +1771,20 @@ \subsubsection{Inputs}\label{inputs-21-003}
 
 \subsection{Site:Precipitation}\label{siteprecipitation}
 
-The Site:Precipitation object is used to describe the amount of water precipitation at the building site over the course of the simulation run period. Precipitation includes both rain and the equivalent water content of snow. Precipitation is not yet described well enough in the many building weather data files. So this object can be used to provide the data using Schedule objects that define rates of precipitation in meters per hour.
+The Site:Precipitation object is used to describe the amount of water
+precipitation at the building site over the course of the simulation run period.
+Precipitation includes both rain and the equivalent water content of snow.
+Precipitation is not yet described well enough in the many building weather data
+files. So this object can be used to provide the data using Schedule objects
+that define rates of precipitation in meters per hour. In 9.6 and later
+versions, Site:Precipitation is also used in the determination of the
+rain indicator during the run period. As the rain fall rate is determined by
+Site:Precipitation when it is defined. Otherwise, it takes value from liquid
+precipitation from the epw. During the run period, when the rain rate is
+no less than 0.8 mm per hour, then the rain flag is overwritten to true.
+This overwritten occurs during the run period when there is only one weather
+file data periods and the number of timesteps in an hour is 1. During the sizing
+period, the rain indicator defined in the SizingPeriod:DesignDay will be used instead.
```

Comment:
... versions, Site::Precipitation is also used to set the rain indicator. During a weather file simulation (RunPeriod), if Use Weather File Rain Indicators is Yes and the precipitation rate is no less than 0.8 mm per hour, then the rain flag is set to true. During a sizing simulation the rain indicator defined in SizingPeriod:DesignDay, SizingPeriod:WeatherFileDays, or SizingPeriod:WeatherFileConditionType will be used instead.